docs/BreakingChanges/046 EventListener truncates strings with embedded null.md

```diff
@@ -0,0 +1,25 @@
+## 46: EventListener truncates strings with embedded nulls
```

Comment:
{
  "Id": "46",
  "Title": "EventListener truncates strings with embedded nulls",
  "Details": "EventListener truncates strings with embedded nulls. Null characters are not supported by the EventSource class. The change only affects apps that use EventListener to read EventSource data in process and that use null characters as delimiters.",
  "Suggestion": "EventSource data should be updated, if possible, to not use embedded null characters.",
  "Link": "https://msdn.microsoft.com/en-us/library/dn458360%28v=vs.110%29.aspx#Core",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Diagnostics.Tracing.EventListener.#ctor",
    "M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel)",
    "M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel,System.Diagnostics.Tracing.EventKeywords)",
    "M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel,System.Diagnostics.Tracing.EventKeywords,System.Collections.Generic.IDictionary{System.String,System.String})"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Diagnostics.Tracing.EventListener.#ctor
- [x] M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel)
- [x] M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel,System.Diagnostics.Tracing.EventKeywords)
- [x] M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel,System.Diagnostics.Tracing.EventKeywords,System.Collections.Generic.IDictionary{System.String,System.String})